sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.getAllInProgressApplications.e2e-spec.ts

```diff
@@ -0,0 +1,204 @@
+  getAuthRelatedEntities,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+import {
+  createFakeInstitutionLocation,
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeApplicationOfferingRequestChange,
+} from "@sims/test-utils";
+import {
+  ApplicationOfferingChangeRequestStatus,
+  InstitutionLocation,
+} from "@sims/sims-db";
+import { FieldSortOrder, getISODateOnlyString } from "@sims/utilities";
+import { getUserFullName } from "../../../../utilities";
+
+describe("ApplicationOfferingChangeRequestAESTController(e2e)-getAllInProgressApplications", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeFLocation: InstitutionLocation;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation({ institution: collegeF });
+  });
+
+  it("Should return all in progress application offering change requests when requested.", async () => {
+    // Arrange
+
+    // Student 1 has an in progress with student application offering change request for College F.
+    const inProgressWithStudentApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(db, {
+        institutionLocation: collegeFLocation,
+      });
+    // Student 2 has an in progress with SABC application offering change request for College F.
+    const inProgressWithSABCApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          },
+        },
+      );
+    // Student 3 has an in progress with student application offering change request with a different institution.
+    const inProgressWithStudentApplicationOfferingChangeWithDifferentInstitution =
+      await saveFakeApplicationOfferingRequestChange(db);
+    // Student 4 has an completed application for College F, that has an approved application offering change request.
+    const approvedApplicationOfferingChange =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          institutionLocation: collegeFLocation,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.Approved,
+          },
+        },
+      );
+
+    const applicationApprovedApplicationOfferingChange =
+      approvedApplicationOfferingChange.application;
+    const applicationInProgressWithStudentApplicationOfferingChangeWithDifferentInstitution =
```

Comment:
Do we need to create data for different institution for aest test case?

Reply:
I thought it would be good to have explicitly different institutions in the test to show we display all institutions but that caused more confusion so I removed any references for institutions in the test.